package.json

```diff
@@ -463,7 +463,7 @@
         "r.sessionWatcher": {
           "type": "boolean",
           "default": false,
-          "description": "Enable R session watcher (experimental)"
+          "description": "Enable R session watcher (experimental). Restart required to take effect"
```

Comment:
Looks good but do we need `.` in the end?

Reply:
All vscode-R's setting descriptions end without a `.` including the multi-sentence ones (although I think I wrote the only other multi-sentence one). Looking through the settings provided by VS Code itself it seems that the standard style is to end all setting descriptions with a `.`. I'm happy to change *all* our setting descriptions to end in `.` for consistency. I'll add a commit to do that.

Reply:
Currently, we do not have `.`.
 But, other major extensions like Python and ESlint use a `.`.
it is a good time to use popular format.

Reply:
@Ikuyadeu Thanks for the thumbs up on the style change!

Reply:
I take a look at vscode settings description and it looks like most have ending `.`, and the rule could be as simple as:

1. If the description is only one sentence, vscode settings descriptions uses ending `.` but some extension do not.
2. If the description is multiple sentences, a ending `.` seems always used.

To be consistent with vscode and most popular extensions, we could always use `.`.